src/components/QLoader/QLoader.vue

```diff
@@ -1,17 +1,36 @@
 <template>
-  <span class="q-loader spinner" :style="`border-top-color: ${borderTopColor}`" />
+  <span class="q-loader spinner" :style="`border-color: ${bgColor}; border-top-color: ${accentColor}; height: ${size}px; width: ${size}px; border-width: ${borderWidth}px`" />
```

Comment:
Please change like this : https://github.com/quiches-group/quichtify/blob/master/src/components/QTabs/QTabs.vue

Reply:
Impossible to make it work so I leave it like that...